lib/main_containers.pm

```diff
@@ -31,36 +31,41 @@ sub is_container_test {
     get_var('CONTAINER_RUNTIME') ? return 1 : return 0;
 }
 
+sub is_container_image_test {
+    my $flavor = get_required_var('FLAVOR');
+    return $flavor =~ /(Container-Image|NET)/;
```

Comment:
In Leap image tests, it's scheduled as part of the `JeOS` flavors, e.g. https://openqa.opensuse.org/tests/1942080

Reply:
I'm not sure how to solve this... we have 
```
if (is_jeos)
    load_jeos_tests();
```
and then if I load the container tests, it will have run first the specific jeos tests...